core/node/fee_model/src/l1_gas_price/gas_adjuster/mod.rs

```diff
@@ -134,7 +133,6 @@ impl GasAdjuster {
                 "Effective gas price is too high: {gas_price}, using max allowed: {}",
                 max_l1_gas_price
             );
-            KEEPER_METRICS.gas_price_too_high.inc();
```

Comment:
We've discussed in the past that this metric isn't used anywhere and it's OK to remove it (until we feel that we need it).